src/client/components/Swipe/InfoPanel/InfoPanel.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import SlidingPanel from 'react-sliding-panel';
+
+const SwipeCard = props => {
+  const { place, isModalShown, setIsOpen } = props;
+  const { image } = place;
+
+  const handleClick = (showInfo) => {
+      setIsOpen(showInfo);
+  };
+
+  return (
+    <div className="info-panel">
+      <SlidingPanel
+        type={"bottom"}
+        isOpen={isModalShown}
+        closeFunc={() => handleClick(false)}
+      >
+        <div className="info-content">
+          <button className="info-button-close" onClick={() => handleClick(false)}>
+            <span className="text">tap here to close</span>
+            <span className="icon"></span>
+          </button>
+          <div  className="info-title">
+            { place.name + ", " + place.price }
+          </div>
+          <div  className="info-intro">
+            { place.desc }
```

Comment:
![image](https://user-images.githubusercontent.com/15785697/67153501-9b9a5d80-f2b8-11e9-8421-c6b8ef2333a4.png)

there's something weird here. the text is cut. i can't look into it right now, but give it a shot. if you still can't fix it, let me know, we can look into it in more details!